Gems/EMotionFX/Code/EMotionFX/Source/AnimGraph.cpp

```diff
@@ -902,6 +902,7 @@ namespace EMotionFX
 
     void AnimGraph::OnRetargetingEnabledChanged()
     {
+        SetDirtyFlag(true);
```

Comment:
Can we call AnimGraph::SetRetargetingEnabled(true); here instead and move the SetDirtyFlag(true) call to the SetRetargetingEnabled function?